python-packages/sra_client/tox.ini

```diff
@@ -8,3 +8,17 @@ deps=-r{toxinidir}/requirements.txt
 commands=
    nosetests \
       []
+
+[testenv:run_tests_against_test_deployment]
+commands =
+    # install dependencies from real PyPI
+    pip install mypy_extensions pytest
+    # install package-under-test from test PyPI
+    pip install --index-url https://test.pypi.org/legacy/ 0x-sra-client
```

Comment:
Why using `/legacy`? Maybe add a comment.

Reply:
More trouble/noise than it's worth, in my opinion.  This is the standard boilerplate you'll find if you google how to publish to the test PyPI.  If you're curious, there's https://github.com/pypa/warehouse/issues/2285 , but it was tl/dr for me. :stuck_out_tongue: